### PR TITLE
Code quality

### PR DESCRIPTION
**Pull request by colinodell**
Unit tests to increase code coverage and overall quality.

Also contains a couple minor changes noticed while writing/executing the tests.

### CODE REVIEW

**Comment by colinodell** [comment]
This boosts code coverage from 92% to 96%! :tada:

File: src/Inline/Element/AbstractWebResource.php [hunk 1]
```diff
@@ -33,16 +33,4 @@ public function getUrl()
     {
         return $this->url;
     }
-
-    /**
-     * @param mixed $url
-     *
-     * @return $this
-     */
-    public function setUrl($url)
```
Comment (GrahamCampbell) [inline src/Inline/Element/AbstractWebResource.php]: breaking change?

File: src/Inline/Element/AbstractInlineContainer.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 namespace League\CommonMark\Inline\Element;
 
-class AbstractInlineContainer extends AbstractInline
+abstract class AbstractInlineContainer extends AbstractInline
```
Comment (GrahamCampbell) [inline src/Inline/Element/AbstractInlineContainer.php]: breaking change?

Comment (GrahamCampbell) [inline src/Inline/Element/AbstractInlineContainer.php]: nvm, i see the branch alias has already been changed :)

Comment (colinodell) [inline src/Inline/Element/AbstractInlineContainer.php]: :)

**Pull request merged**